### DISCUSSION TITLE
No sensor information in merged tours?

### DISCUSSION THREAD

**Discussion by ovahead** [mytourbook/mytourbook#1155] [General]
Is it actually known or even wanted that the information of the sensors gets lost in the tours put together by MyTourbook? ;)

![Sensordaten](https://github.com/mytourbook/mytourbook/assets/112829463/222f770e-2c38-4bb6-aa63-d58b01b7df0f)

**Comment by wolfgang-ch** [comment]
> ... in the tours put together by MyTourbook?

Which action are you using?

**Reply by ovahead** [reply]
For that I used Connect Tours from the context menu of the Tour Import view.
As you can see there is also no information about the Batterie Charge...

**Reply by wolfgang-ch** [reply]
> For that I used Connect Tours from the context menu of the Tour Import view.

There is no "Connect Tours" in the context menu, are you using "Concatenate Tours..."?

![image](https://github.com/mytourbook/mytourbook/assets/1283445/14ab1154-c976-49ce-9831-bd36f1721386)

**Reply by ovahead** [reply]
Yes must be, sorry. I'm using the german version of MT. So it's "Touren verbinden..."

**Comment by wolfgang-ch** [comment]
>Is it actually known or even wanted that the information of the sensors gets lost in the tours put together by MyTourbook? ;)

How should sensor values be concatenated, these are not dataseries e.g. distance or elevation?

**Reply by ovahead** [reply]
How should I know that? ;)
The only thing I noticed was that sensor values from merged tours with FIT File Repair is displayed in MT. That's why I thought it would be normal for this data to be displayed.
But if it's possible from a single tour, why shouldn't it be possible from merged tours as well?

![FFRT-Merged-Tours](https://github.com/mytourbook/mytourbook/assets/112829463/4cc8745a-dc06-42e6-a9b6-d7275b5d6d95)

**Reply by wolfgang-ch** [reply]
Can you provide sensor value screenshots of a merged tour and the tours which are merged. Then we can see how these values are merged.

**Reply by ovahead** [reply]
Hmm, not really put together.
Battery state of charge and data from the Garmin sensor may only have been taken over from tour 2, but the data from the Magene sensor from tour 1??
![Tour1](https://github.com/mytourbook/mytourbook/assets/112829463/13b6d204-c317-4922-8dbc-d4a7613a2e28)
![Tour2](https://github.com/mytourbook/mytourbook/assets/112829463/fad86c9c-87fc-4f96-af6b-696db58e8ba4)

This are screenshots from the tours which were merged and in my last post is the screenshot from the merged tour.
